draft-irtf-cfrg-hash-to-curve.md

```diff
@@ -831,134 +831,207 @@ Instantiating one of these protocols with an elliptic curve group motivates
 the term "hashing to the curve", i.e., encoding bitstrings to points on an
 elliptic curve.
 
-The hash2curve(alpha) function can be easily constructed by
-using the output of a cryptographically secure hash function H as the input of
-an encoding function.
-On the one hand, hash2curve is difficult to invert since it is computationally
-intractable to produce an input alpha that maps to hash2curve(alpha) due to H
-is pre-image resistant.
+One can easily construct a hash to curve function h(x) by using the output of a
+cryptographically secure hash function H as the input of an encoding function.
+On the one hand, h(x) is difficult to invert since it is computationally
+intractable to produce an input x that maps to h(x) due to H is pre-image
+resistant.
 On the other hand, the uniformity property is not met as the output of an
 encoding is distinguishable from a random distribution. Hence, using
-hash2curve(alpha) is not sufficient to get uniformity, however it can be used
-as a building block for obtaining a random oracle as is described in {{rom}}.
+h(x) is not sufficient to get uniformity and cannot be used as a random oracle.
+
+Brier et al. {{BCIMRT10}} describe two generic constructions that give a
+hash function approximating a random oracle. Farashahi et al. {{FFSTV13}}
+and Tibouchi and Kim {{TK17}} refine this analysis. In particular, Farashahi
+et al. show that summing two independent evaluations of an injective encoding
+suffices to approximate a random oracle to an elliptic curve. This construction
+is given in {{roadmap}}.
+
+# Roadmap {#roadmap}
+
+A general framework for mapping bit strings into points of an elliptic curve is
+presented. The construction of these mappings relies on three basic functions.
```

Comment:
Nit: passive voice in first sentence. Maybe:

This section presents a general framework for mapping bit strings into points on an elliptic curve. To construct these mappings, we rely on three basic functions: